server/internal/core/application/covenant.go

```diff
@@ -114,18 +115,43 @@ func (s *covenantService) Stop() {
 	close(s.onboardingCh)
 }
 
-func (s *covenantService) SpendVtxos(ctx context.Context, inputs []domain.VtxoKey) (string, error) {
-	vtxos, err := s.repoManager.Vtxos().GetVtxos(ctx, inputs)
+func (s *covenantService) CreateReverseBoardingAddress(ctx context.Context, userPubkey *secp256k1.PublicKey) (string, error) {
+	return "", fmt.Errorf("unimplemented")
```

Comment:
giving the impact of this kind of boarding on liquidity requirements i think it does make really sense to have it implemented also on the covenant version in this pr